### PR TITLE
Tp 8

### PR DESCRIPTION
**Pull request by Brandon-B9845**
Made the map to generate the Card component.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -67,7 +77,10 @@ export default function Home() {
           <div className='pointsOfInterest-filter-container' >
             <Button name='info-sidebar-btn' handler={infoSidebarHandler} value='Holder Value'/>          
           </div>
-          {showInfo && <div className='info-sidebar'></div>}
+          {showInfo && <div className='info-sidebar'>
+            {placesInfo.map((place: any) => <Card  id={place.place_id} name={place.name} rating={place.rating} price={place.price_level} address={place.formatted_address} ratings_number={place.user_ratings_total} />)}
```
Comment (CSpiegelhalter) [inline pages/index.tsx:81]: [pebble] So this file is becoming pretty big. The side bar thing should be broken out into its own component for manageability and reusability

File: pages/index.tsx [hunk 2]
```diff
@@ -45,13 +49,19 @@ export default function Home() {
   //sets the space where the map should be to loading... if it is not yet rendered
   if (!isLoaded) return <div>Loading...</div>
 
+  let test: any
```
Comment (CSpiegelhalter) [inline pages/index.tsx:52]: kill this plz

File: pages/index.tsx [hunk 3]
```diff
@@ -45,13 +49,19 @@ export default function Home() {
...
+  let test: any  
   // Sidebar handler
   //This is the function that calls the PointsOfInterest api and flips the state to show or not show the information sidebar
   const infoSidebarHandler = async () => {
     const data = await fetch('/api/pointsOfInterest')
-    console.log(await data.json())
+    setPlacesInfo(await data.json())
     setShowInfo(!showInfo)
+    setPlacesInfo(prevVal => prevVal.results)
+      console.log(placesInfo)
```
Comment (CSpiegelhalter) [inline pages/index.tsx:60]: kill this too

**Review by CSpiegelhalter — CHANGES_REQUESTED** [review]
Make the side bar its own component

File: pages/index.tsx (hunk 2 above)
Comment (Brandon-B9845) [reply pages/index.tsx:52]: It was my only friend...

File: pages/index.tsx (hunk 3 above)
Comment (Brandon-B9845) [reply pages/index.tsx:60]: Any last words?

File: pages/index.tsx (hunk 1 above)
Comment (Brandon-B9845) [reply pages/index.tsx:81]: Got you fam

**Review by CSpiegelhalter — APPROVED** [review]

**Pull request merged**